woodwork/logical_types.py

```diff
@@ -31,6 +31,19 @@ def __str__(self):
         return str(self.__class__)
 
 
+class Age(LogicalType):
+    """Represents Logical Types that contain positive numbers. Has 'numeric' as a standard tag.
+
+    Examples:
+        .. code-block:: python
+
+            [15, 22, 45]
+            [30, 62, 87]
+    """
+    primary_dtype = 'Int64'
```

Comment:
Does having the nullable dtype here mean we need to have a corresponding `AgeNullable` logical type as well, or is that just because of the type inference that occurs with Integer that we need to allow for inferring with and without nullable dtypes? @gsheni @thehomebrewnerd

Reply:
@tamargrey that's an interesting point and something I hadn't thought of. 

Perhaps to keep consistency, we should have Age (int64) and AgeNullable (Int64). What do you think @jeff-hernandez @tamargrey ?

Reply:
I think it makes sense to have a corresponding nullable type for consistency. When would a user want `Age` specifically instead of `AgeNullable`? I kind of thought the nullable types were a superset of the non-nullable types.

Reply:
We have 1 other Logical Type that has Int64 as the dtypes (IntegerNullable). If a user does not have any NaNs we use int64 and Integer. 
This Age Logical Type will be another Int64/int64 dtype and so we want to get a similar behavior has Integer. 

If a user has NaNs, they use AgeNullable and if they don't they use Age.

Reply:
I think I'm okay with having an `AgeNullable`, but we might want to have something in the docstrings that indicate that Age won't handle null values. Since with boolean and integer, you get different inference, but with Age you have to know to use it.

Reply:
Sounds good. I'll add `AgeNullable` to this PR.